runtime/Python2/src/antlr4/atn/ATNConfig.py

```diff
@@ -140,7 +140,7 @@ def __eq__(self, other):
             return False
         if self.passedThroughNonGreedyDecision != other.passedThroughNonGreedyDecision:
             return False
-        if self.lexerActionExecutor is not other.lexerActionExecutor:
+        if not(self.lexerActionExecutor == other.lexerActionExecutor):
```

Comment:
That's a trick between Python2 and Python3 : Python3 doesn't expose that behaviour, Python2 does not call the `__eq__` function when using the `!=` operator.

```python
class a(object):
    def __init__(self, v):
        self.v = v
    def __eq__ (self, other):
        return other.v == self.v

o1 = a(42)
o2 = a(73)
o3 = a(42)

print("o1==o2:", o1==o2)
print("o1==o3:", o1==o3)
print("o1!=o2:", o1!=o2)
print("o1!=o3:", o1!=o3)
```

If you execute that code with python 2 and python 3, you'll see a different output for o1 != o3 : python 2 will say True while python 3 will say False.
In Python2, one is supposed to implement also the `__ne__` function. For consistency, I chose to use the existing `__eq__` operators in both python 2 and python 3.